charts/jupyterhub/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+name: jupyterhub
+version: 0.0.1
+appVersion: 1.1.0
+description: Nublado2 JupyterHub installation
+home: https://github.com/lsst-sqre/nublado2
+maintainers:
+  - name: cbanek
+sources:
+  - https://github.com/lsst-sqre/nublado2
+  - https://github.com/lsst-sqre/charts
+kubeVersion: '>=1.11.0-0'
```

Comment:
I thought this chart was just a copy of the upstream one? This looks very specific to us. Shouldn't we use the same upstream `Chart.yaml` file?

Reply:
Can't do it because the upstream chart.yaml has a bunch of stuff in it that is updated by their CI, same for some of the values.yaml where the images are put in

Reply:
Yeah exactly that - if you look over here https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/jupyterhub/Chart.yaml at the chart there's a bunch of "set by chartpress" stuff, which is their own CI and seems to get replaced by templating.  I'd honestly rather push my change back upstream but wasn't sure if there would be other changes and how long it might take.